build/tools/amebad/gnu_utility/README.md

```diff
@@ -2,28 +2,35 @@
 
 ### Preparations
 
-1. Prepare J-Link with firmware upgraded to V10.
-2. Install JLinkGDBServer.
-3. Set 'environment variable' for JLinkGDBServer if not done so.
-
-=============================
+1. Prepare J-Link and Install JLinkGDBServer.  
+Download JLinkGDBServer from this [link](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).  
+After download, install through below command.  
+```
+$ sudo dpkg -i JLink_Linux_V686g_x86_64.deb
+```
+
+2. Upgrade the firmware to V10.  
+In 'GDB Server' window, there is firmware version like below.
```

Comment:
How about below?
Upgrade the firmware to V10 and verify through the message of 'GDB Server' window like below.